packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -260,7 +260,7 @@ export class LitDevTutorial extends LitElement {
   };
 
   private _writeUrl() {
-    window.history.pushState(
+    window.history.replaceState(
```

Comment:
Thanks for the PR!

I don't think this line is the problem. We want this method to use `pushState`, because we want to add a history entry when you move to the next or previous tutorial step, which is what this method does.

I think the problem is actually in playground itself here: https://github.com/google/playground-elements/blob/05538da0ee6fe13b3345d89b805dd06b2b8f30b3/src/playground-preview.ts#L198

When an iframe reloads, the parent window's history is affected. So the history states are actually from the playground preview reloading. I think the trick to prevent that would be to remove the iframe from the DOM before we change its `src`, and then attach it after.

Reply:
Ah that makes sense. I had some trouble getting the project to build so I thought I'd raise this PR to bring this problem to attention. 

```
When an iframe reloads, the parent window's history is affected. So the history states are actually from the playground preview reloading. I think the trick to prevent that would be to remove the iframe from the DOM before we change its src, and then attach it after.
```
If that's the case, I'll look into it after I've got the project set up

Reply:
I'll open up a separate PR for that as I get around to doing it. Closing this one as it has unnecessary commits